tests/Api/Responses/shop/order_item/get_order_item_by_guest.json

```diff
@@ -1,6 +1,6 @@
 {
     "@context": "\/api\/v2\/contexts\/OrderItem",
-    "@id": "\/api\/v2\/shop\/order-items\/@integer@",
+    "@id": "\/api\/v2\/shop\/orders\/token\/items\/@integer@",
```

Comment:
Below, there is an `order` key, that could be changed from:
`"order": "\/api\/v2\/shop\/orders\/@string@"`
to
`"order": "\/api\/v2\/shop\/orders\/token"`.